ledger/participant-integration-api/src/test/suite/scala/platform/store/cache/InMemoryFanoutBufferSpec.scala

```diff
@@ -208,57 +272,106 @@ class EventsBufferSpec extends AnyWordSpec with Matchers with ScalaCheckDrivenPr
   "prune" when {
     "element found" should {
       "prune inclusive" in withBuffer() { buffer =>
+        buffer.lookup(txAccepted1.transactionId) shouldBe Some(txAccepted1)
```

Comment:
Could be shorted to `verifyPresent(buffer, Seq(txAccepted1, txAccepted2, txAccepted3, txAccepted4)` ?

Reply:
thanks. Done